.gitignore

```diff
@@ -74,3 +74,5 @@ MacOSX*
 *report.xml
 *report.json
 *.out
+
+contracts/yarn.lock
```

Comment:
Isn't this required to keep the build reproducible?

Reply:
No, we have `pnpm-lock.yaml` for that. Each npm package manager has their own lockfile format:

- npm uses package-lock.json
- yarn uses yarn.lock
- pnpm uses pnpm-lock.yaml

We use `pnpm` now :)

Reply:
![6368ab58-8a44-4749-a5b1-bf5f6aab5c04_text.gif](https://user-images.githubusercontent.com/1194128/227392411-35af1467-9e56-48a2-857e-20f2dae3febe.gif)